game_patch/multi/server.cpp

```diff
@@ -197,12 +243,83 @@ void load_additional_server_config(rf::Parser& parser)
     }
 }
 
+// memory addresses for weapon stay exemption indexes
+constexpr std::pair<bool WeaponStayExemptionConfig::*, uintptr_t> weapon_exemptions[] = {
+    {&WeaponStayExemptionConfig::rocket_launcher, 0x00872458},
+    {&WeaponStayExemptionConfig::heavy_machine_gun, 0x00872460},
+    {&WeaponStayExemptionConfig::sniper_rifle, 0x00872440},
+    {&WeaponStayExemptionConfig::assault_rifle, 0x00872470},
+    {&WeaponStayExemptionConfig::machine_pistol, 0x0085CCD8},
+    {&WeaponStayExemptionConfig::shotgun, 0x00872108},
+    {&WeaponStayExemptionConfig::scope_assault_rifle, 0x0087245C},
+    {&WeaponStayExemptionConfig::grenade, 0x00872118},
+    {&WeaponStayExemptionConfig::remote_charge, 0x0087210C},
+    {&WeaponStayExemptionConfig::handgun, 0x00872114},
+    {&WeaponStayExemptionConfig::flamethrower, 0x0087243C},
+    {&WeaponStayExemptionConfig::riot_stick, 0x00872468},
+    {&WeaponStayExemptionConfig::riot_shield, 0x0085CCE4},
+    {&WeaponStayExemptionConfig::rail_gun, 0x00872124}};
+
+// declare optional vector for weapon stay exemptions
+std::optional<std::vector<uintptr_t>> weapon_stay_exempt_indexes;
```

Comment:
what's the point of making it an optional? Empty vector would have the same effect